tests/phpunit/tests/option/option.php

```diff
@@ -359,16 +359,85 @@ public function test_option_autoloading( $name, $autoload_value, $expected ) {
 	 */
 	public function data_option_autoloading() {
 		return array(
-			array( 'autoload_yes', 'yes', 'yes' ),
-			array( 'autoload_true', true, 'yes' ),
-			array( 'autoload_string', 'foo', 'yes' ),
-			array( 'autoload_int', 123456, 'yes' ),
-			array( 'autoload_array', array(), 'yes' ),
-			array( 'autoload_no', 'no', 'no' ),
-			array( 'autoload_false', false, 'no' ),
+			array( 'autoload_yes', 'yes', 'on' ),
+			array( 'autoload_true', true, 'on' ),
+			array( 'autoload_string', 'foo', 'auto' ),
+			array( 'autoload_int', 123456, 'auto' ),
+			array( 'autoload_array', array(), 'auto' ),
```

Comment:
See above, those should result in the same positive outcome as they did so far, which should be the case with my proposed change above.
```suggestion
			array( 'autoload_string', 'foo', 'on' ),
			array( 'autoload_int', 123456, 'on' ),
			array( 'autoload_array', array(), 'on' ),
```